src/autobridgecalculator.js

```diff
@@ -38,7 +38,7 @@ AutobridgeCalculator.NULL_AMOUNT = Utils.normalizeAmount('0');
  * @return {Array}
  */
 
-AutobridgeCalculator.prototype.calculate = function() {
+AutobridgeCalculator.prototype.calculate = function(callback = function() {}) {
```

Comment:
Why is the callback optional? Is there a way to get the results without using the callback?